docs/source/get_involved/code_of_conduct.rst

```diff
@@ -342,14 +397,161 @@ Guide <https://www.djangoproject.com/conduct/>`__ and we are also
+* accidental violations of CoC causally due to protected characteristics of the person violating the CoC
+* systemic discrimination issues whose resolution would cause discomfort in the community
+
+Guidance rules:
+
+* the less powerful party should be protected, i.e., CoC cannot be invoked by lead community member to silence critique of how funds are spent.
+  But, CoC should protect new contributors or early career members from more powerful community members, e.g., in operational or technical disputes.
+* technical content is not invalidated by tone, but tone must remain civil at all times.
+  I.e., valid arguments are not discarded because of tone, but participants in a technical
+  dispute may be punished for tone that is unwelcoming, e.g., name calling etc.
+* conditions that make compliance with certain aspects of CoC more difficult should be taken into account,
+  especially in parties that are the less powerful in a dynamic. Common examples are certain forms of neuroatypicality.
+  A more illustrative but very rare example would be Tourette's with the rare swearing tic that cannot be "turned off".
+* issues of systemic exclusion or discrimination should be addressed when apparent, even at the cost of some community discomfort.
+  E.g., meetings should be moved to less convenient times, or processes made more "formal",
+  if not doing so would exclude others in a discriminatory fashion.
+
+Behaviour that is clearly racist, sexist, etc, is always a CoC violation, and never part of trade-offs.
+
+
+A.2 Guidance on applying the CoC for resource allocation decisions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This section deals with resource allocation decisions by ``sktime`` community members,
+and criteria for decision making on resource allocation violating the code of conduct.
+
+Scope
+^^^^^
+
+In scope for this section are all resource allocation decisions that satisfy at least one of the following criteria:
+
+* any decisions that allocate resources primarily obtained due to the ``sktime`` project and the volunteer contributions of the community.
+  This includes research grants awarded directly for the benefit of the ``sktime`` project,
+  commercial opportunities arising from or directly directed to the ``sktime`` project,
+  and any activities that use the ``sktime`` brand as a primary branding.
+  This condition applies even if relevant decisions are made in personal roles, or roles other than ``sktime`` official roles.
+* any decisions by members of the ``sktime`` developer community, in their execution of an official ``sktime`` role.
+  This especially includes core developer, Community Council, and Code of Conduct Committee roles.
+
+Examples for in-scope:
+
+* an academic allocating grant monies from a grant, or a data science consultant providing consultancy services, with a primary ``sktime`` branding
+* hiring decisions for roles that are advertised with a primary ``sktime`` branding
```

Comment:
First two examples are out-of-scope. Anyone can request money for their grant claiming they're going to post their research results in sktime. Anyone can open a business that sells forecasts and put the "we're widely using sktime", the way now people put "we use OS libraries like statsmodels and pandas".

Reply:
Not true: the BSD-3 license puts ``sktime`` branded activities within scope of license holder approval.

The key property is being ``sktime`` branded e.g., under a company name like "``sktime`` consulting", or a random university founding an ``sktime`` department.

Whereas a company named "``foo`` consulting", or a random university's "forecasting toolbox department" would not be impacted.

Reply:
the bullet points (due to the technical term "``sktime`` branded") apply to, and only to, activities that directly and prominently use the ``sktime`` name. They do not apply to similar activities that do not use the ``sktime`` name.

Is that subtlety clear?

To repeat: we have *legal* decision making power - supported almost worldwide by national and international legal systems! - on anything called ``sktime``, due to the BSD-3 license we are using.

Whereas the CoC has jurisdiction only where we can enforce it, i.e., among elected members.

Reply:
"Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission." - I think that doesn't mean that people can't use sktime in the name of their project, etc, is that correct? Or did you mean some other part of the license that I missed?

Reply:
> I think that doesn't mean that people can't use sktime in the name of their project, etc, is that correct?

I think it does mean that.
No one can use ``sktime`` in their project or product, without permission, it does mean that.

This applies even without a trademark afaik, although the ease with which it can be legally enforced is impacted by whether it is registered.